tests/testthat/test-convertToFlowCore.R

```diff
@@ -0,0 +1,165 @@
+  # LogScale:
+  expect_equal(tl@transforms$`FSC-H`@f(400), log10(400)) # nolint
+  # LinearScale:
+  expect_equal(tl@transforms$`SSC-H`@f(10), 10) # nolint
+  # ArcSinhScale:
+  expect_equal(tl@transforms$`Ax488-A`@f(10), asinh(10 / 300)) # nolint
+  # ArcSinhScale (ensure cofactor captured correctly):
+  expect_equal(tl@transforms$`PE-A`@f(10), asinh(10 / 150)) # nolint
+})
+
+test_that("toFlowCore converts a ScaleSet", {
+  skip_if_not_installed("flowCore")
+  library("flowCore")
+
+  scaleData <- '[{"_id":"5d2f8b4b21fd0676fb3a6a8c","experimentId":"5d2f8b4b21fd0676fb3a6a70","name":"Scale Set 1","scales":[{"channelName":"FSC-H","scale":{"type":"LogScale","minimum":1,"maximum":100000}},{"channelName":"SSC-H","scale":{"type":"LinearScale","minimum":1,"maximum":262144}},{"channelName":"SSC-A","scale":{"type":"ArcSinhScale","minimum":-200,"maximum":262144,"cofactor":150}},{"channelName":"Ax488-A","scale":{"type":"ArcSinhScale","minimum":-200,"maximum":262144,"cofactor":300}},{"channelName":"PE-A","scale":{"type":"ArcSinhScale","minimum":-500,"maximum":262144,"cofactor":150}},{"channelName":"PE-TR-A","scale":{"type":"ArcSinhScale","minimum":-200,"maximum":262144,"cofactor":150}},{"channelName":"PerCP-Cy55-A","scale":{"type":"ArcSinhScale","minimum":-200,"maximum":262144,"cofactor":150}},{"channelName":"PE-Cy7-A","scale":{"type":"ArcSinhScale","minimum":-200,"maximum":262144,"cofactor":150}},{"channelName":"Ax647-A","scale":{"type":"ArcSinhScale","minimum":-200,"maximum":262144,"cofactor":150}},{"channelName":"Ax700-A","scale":{"type":"ArcSinhScale","minimum":-200,"maximum":262144,"cofactor":150}},{"channelName":"Ax750-A","scale":{"type":"ArcSinhScale","minimum":-200,"maximum":262144,"cofactor":150}},{"channelName":"PacBlu-A","scale":{"type":"ArcSinhScale","minimum":-200,"maximum":262144,"cofactor":150}},{"channelName":"Qdot525-A","scale":{"type":"ArcSinhScale","minimum":-200,"maximum":262144,"cofactor":150}},{"channelName":"PacOrange-A","scale":{"type":"ArcSinhScale","minimum":-200,"maximum":262144,"cofactor":150}},{"channelName":"Qdot605-A","scale":{"type":"ArcSinhScale","minimum":-200,"maximum":262144,"cofactor":150}},{"channelName":"Qdot655-A","scale":{"type":"ArcSinhScale","minimum":-200,"maximum":262144,"cofactor":150}},{"channelName":"Qdot705-A","scale":{"type":"ArcSinhScale","minimum":-200,"maximum":262144,"cofactor":150}},{"channelName":"Time","scale":{"type":"LinearScale","minimum":1,"maximum":262144}}],"__v":1,"updated":"2020-10-27T18:38:55.554Z"}]' # nolint
+  ss <- jsonlite::fromJSON(scaleData)
+  tl <- toFlowCore(ss)
+  expect_equal(tl@transformationId, ss$name)
+  # LogScale:
+  expect_equal(tl@transforms$`FSC-H`@f(400), log10(400)) # nolint
+  # LinearScale:
+  expect_equal(tl@transforms$`SSC-H`@f(10), 10) # nolint
+  # ArcSinhScale:
+  expect_equal(tl@transforms$`Ax488-A`@f(10), asinh(10 / 300)) # nolint
+  # ArcSinhScale (ensure cofactor captured correctly):
+  expect_equal(tl@transforms$`PE-A`@f(10), asinh(10 / 150)) # nolint
+})
```

Comment:
🧐 is this identical to the test just above?

Reply:
The only difference is that it calls `toFlowCore` instead of `scaleSetToTransformList` (public vs private api)

Reply:
The private API is 100% covered by the public API, right? If so, ✂️

Reply:
:+1: